src/theme/GlobalStyles.js

```diff
@@ -0,0 +1,9 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyles = createGlobalStyle`
+    body {
+        margin: 0;
```

Comment:
Also, I would add `padding: 0` and `box-sizing: border-box`

```suggestion
        margin: 0;
        padding: 0;
        box-sizing: border-box;
```

Reply:
I thought about add something like this [styled-normalize](https://github.com/sergeysova/styled-normalize). What do you think?

Reply:
Why do you want it? I think we can achieve the same result without it and additional tags in code like this:
```
<React.Fragment>
    <Normalize />
    <App />
</React.Fragment>
```

Reply:
Ok, we can do it ourselves

Reply:
I looked at that github but i didn't see any pros of that solution